recipes/imglyb/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "imglyb" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 99f492903a9abe03056867d3e098917bdb1baf3ddf2fa56b6fa243d93c219ccf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
```

Should we install with `--no-deps` flag?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv  --no-deps"
```